model/src/lib.rs

```diff
@@ -56,6 +56,25 @@ impl OrderCreation {
     pub fn token_pair(&self) -> Option<TokenPair> {
         TokenPair::new(self.buy_token, self.sell_token)
     }
+    pub fn order_owner(&self) -> H160 {
+        // dummy implementation, waiting for Valentins PR
+        H160::zero()
+    }
+    pub fn order_digest(&self) -> H256 {
+        // dummy implementation, waiting for Valentins PR
+        H256::zero()
+    }
+    pub fn order_uid(&self) -> OrderUid {
+        let owner = self.order_owner();
+        let digest = self.order_digest();
+        let valid_to = self.valid_to.to_be_bytes();
+        OrderUid(
```

Comment:
Ubernit: I know we are also doing this in contracts but ID should suffice as meaning unique ID. Since that what an ID is. Explicitly stating uniqueness is redundant in my opinion.

Reply:
I don't have a strong opinion on the name but I do strongly feel that it should be the same name as in contracts. In contracts it is called uid (as you also mentioned) so we should keep the same name.

Reply:
I will not change it, unless it is changed in the openAPI documentation PR. If you guys feel strongly about it, please raise it there.